src/views/SignTransactionSuccess.vue

```diff
@@ -16,7 +16,7 @@
 import { Component, Vue } from 'vue-property-decorator';
 import Network from '@/components/Network.vue';
 import { SmallPage } from '@nimiq/vue-components';
-import { SignTransactionResult } from '../lib/RequestTypes';
+import { SignedTransaction } from '../lib/RequestTypes';
```

Comment:
Unused import

Reply:
Actually I forgot to add it as a typecheck when assigning the `result`.